distribution/apim-accelerator/resources/repository/conf/deployment.toml

```diff
@@ -418,6 +418,26 @@ policy.limit.time_unit = "min"
 patient_id_uri = "http://wso2.org/claims/patientId"
 patient_id_key = "patientId"
 
+[[healthcare.backend.auth]]
+# Name of the authentication method. This name must be matched with the Config Name policy attribute in the -
+#  - Replace Backend Auth Token policy.
+name = "epic_pkjwt"
+# Authentication type. Only pkjwt is supported atm.
+auth_type = "pkjwt"
+# External Auth server's Token endpoint URL.
+token_endpoint = "https://localhost:9443/oauth2/token"
+client_id = "client_id"
+private_key_alias = "key_alias"
+is_ssl_enabled = true
```

Comment:
what's the use of this "is_ssl_enabled" ?

Reply:
removed in https://github.com/wso2/healthcare-accelerator/commit/2baf7e5a6442a5828c556972e262416fefba19b3